test/integration/scheduler_perf/config/performance-config.yaml

```diff
@@ -405,11 +405,6 @@
       initNodes: 500
       initPods: 2000
       measurePods: 500
-  - name: 5000Nodes
-    params:
-      initNodes: 5000
-      initPods: 20000
-      measurePods: 5000
```

Comment:
Can you comment them out rather than deleting them? as we may dig out the real timeout cause, and revert them back.

Reply:
Sure.